src/actors/hcd/nbi/nb_simple_actor.jl

```diff
@@ -263,6 +263,10 @@ function _step(actor::ActorSimpleNB)
         ion.power_inside = source1d.total_ion_power_inside
         ion.fast_particles_energy = beam_energy
     end
+    # Add this line to populate fast ion physics
+    IMAS.fast_particles_profiles!(dd)
```

Comment:
`fast_particles_profiles!()` should not be called by the `ActorSimpleNB `, but at the end of the `step()` of the `ActorHCD`. This will add fast ions independently of the NBI, IC actor being run.

Note the comment at the end of `fast_particles_profiles!()` function: **# must decide how to handle quasineutrality`. I suspect this is why this function was not hooked up.

Let's make sure to come up with a plan/system to handle quasi neutrality before merging this PR!